docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -15,9 +15,16 @@ Directories to cache
 
 In your CI's config file, we recommend caching these directories:
 
-- `$HOME/.cache/pants/setup`: the initial bootstrapping of Pants.
-- `$HOME/.cache/pants/named_caches`: caches of tools like pip and PEX.
-- `$HOME/.cache/pants/lmdb_store`: cached content for prior Pants runs, e.g. prior test results.
+- `$HOME/.cache/pants/setup`: The initial bootstrapping of Pants.
+- `$HOME/.cache/pants/named_caches`: Caches used by underlying tools like Pip and PEX.
+
+If you're not using a fine-grained remote caching service, such as the one provided by
+[Toolchain](https://toolchain.com/), the lead sponsor of Pants, then you may also want to
+preserve the local Pants cache:
+
+- `$HOME/.cache/pants/lmdb_store`
+
+However, this directory gets large quickly, and saving and restoring it can become unwieldy.
```

Comment:
```suggestion
However, this directory gets large quickly, so saving and restoring it can become unwieldy. Hence,
remote caching typically offers the best performance.
```